extract/ext/office_extract.py

```diff
@@ -381,30 +382,40 @@ def extract_docx(filename, password_list, output_folder):
         of = olefile.OleFileIO(filename)
     except IOError:
         raise ValueError("Corrupted OLE Document")
+    password = None
 
-    if of.exists("WordDocument"):
-        # Cannot parse these files yet
-        raise ValueError("Legacy Word Document")
-
-    elif of.exists("EncryptionInfo") and of.exists("EncryptedPackage"):
+    # checks new versions of office docs .***x i.e. .xlsx, .docx, etc.
```

Comment:
this doesn't necessarily check for the version of office docs. Rather it checks for the parameters "EncryptionInfo" and "EncryptedPackage" in the OLE File, which based on our testing only exist in new versions of office docs .***x